third_party/move/move-compiler-v2/tests/unused-assignment/unused_assignment.exp

```diff
@@ -0,0 +1,152 @@
+
+Diagnostics:
+warning: Unused parameter `a`. Consider removing or prefixing with an underscore: `_a`
+  ┌─ tests/unused-assignment/unused_assignment.move:2:13
+  │
+2 │     fun foo(a: u8, _b: u8) {
+  │             ^
+
+warning: Unused local variable `y`. Consider removing or prefixing with an underscore: `_y`
+  ┌─ tests/unused-assignment/unused_assignment.move:5:13
+  │
+5 │         let y = 0;
```

Comment:
Note that this warning is generated by @brianrmurphy 's `flow_insensitive_checker`, and NOT by our checker. Upon looking at the generated bytecode below, I believe this assignment has been optimized out on the AST level, and so we can't see it in the bytecode here.